### ISSUE TITLE
Mismatch between annotations and schema

### ISSUE THREAD

**Issue by gravitystorm** [openstreetmap/openstreetmap-website#2235]
I've found a mismatch in the types for various columns, between the schema and the annotations. For example, in changeset_comments, changeset_id and author_id have different types:

```
# == Schema Information
#
# Table name: changeset_comments
#
#  id           :integer          not null, primary key
#  changeset_id :integer          not null
#  author_id    :integer          not null
```

```
CREATE TABLE public.changeset_comments (
    id integer NOT NULL,
    changeset_id bigint NOT NULL,
    author_id bigint NOT NULL,
```

I *think* that it's the annotations need to be updated, but I'm not certain. I created my local database before https://github.com/openstreetmap/openstreetmap-website/commit/b606acba0c28652fa6f066c2b5d989ceae41c194 so I'm being cautious as to which one is incorrect. Any thoughts?

To fix, with a known-good database structure, simply run 'bundle exec rake db:migrate' and it will change the annotations.

**Comment by gravitystorm** [comment]
Ah, it's actually a change in the annotate gem, which showed bigint columns as integers in the version that we used for the first run of annotations. I'm surprised that nobody has encountered this more recently.

**Comment by tomhughes** [comment]
Yes I knew there was something going on there but I hadn't got around to investigating yet...

**Comment by tomhughes** [comment]
Fixed by #2236.
